app/src/debug/java/com/artemzin/qualitymatters/developer_settings/DeveloperSettingsModelImpl.java

```diff
@@ -48,22 +52,24 @@ public DeveloperSettingsModelImpl(@NonNull QualityMattersApp qualityMattersApp,
                                       @NonNull DeveloperSettings developerSettings,
                                       @NonNull OkHttpClient okHttpClient,
                                       @NonNull HttpLoggingInterceptor httpLoggingInterceptor,
-                                      @NonNull LeakCanaryProxy leakCanaryProxy) {
+                                      @NonNull LeakCanaryProxy leakCanaryProxy,
+                                      @NonNull Paperwork paperwork) {
         this.qualityMattersApp = qualityMattersApp;
         this.developerSettings = developerSettings;
         this.okHttpClient = okHttpClient;
         this.httpLoggingInterceptor = httpLoggingInterceptor;
         this.leakCanaryProxy = leakCanaryProxy;
+        this.paperwork = paperwork;
     }
 
     @NonNull
     public String getGitSha() {
-        return BuildConfig.GIT_SHA;
+        return paperwork.get("gitSha");
```

Comment:
So great that we need to change only model for that! No changes in the views/presenters, yay, architecture test passed :)